src/pint/models/binary_dd.py

```diff
@@ -1,23 +1,23 @@
+"""Damour and Deruelle binary model."""
 from pint.models.parameter import floatParameter
 from pint.models.pulsar_binary import PulsarBinary
 from pint.models.stand_alone_psr_binaries.DD_model import DDmodel
 from pint.models.timing_model import MissingParameter
 
 
 class BinaryDD(PulsarBinary):
-    """This is a PINT pulsar binary dd model class a subclass of PulsarBinary.
-    It is a wrapper for independent DDmodel class defined in
-    ./stand_alone_psr_binary/DD_model.py
-    All the detailed calculations are in the independent DDmodel.
-    The aim for this class is to connect the independent binary model with PINT platform
-    DDmodel special parameters:
-    A0 Aberration
-    B0 Aberration
-    GAMMA Binary Einsten delay coeeficient
-    DR Relativistic deformation of the orbit
-    DTH Relativistic deformation of the orbit
+    """Damour and Deruelle binary model.
 
-   Parameters supported:
+    This binary model is described in Damour and Deruelle ????.
```

Comment:
I think this is the ref:
@article{AIHPA_1986__44_3_263_0,
     author = {Damour, Thibault and Deruelle, Nathalie},
     title = {General relativistic celestial mechanics of binary systems. II. The post-newtonian timing formula},
     journal = {Annales de l'I.H.P. Physique th\'eorique},
     pages = {263--292},
     publisher = {Gauthier-Villars},
     volume = {44},
     number = {3},
     year = {1986},
     zbl = {0617.70010},
     mrnumber = {846469},
     language = {en},
     url = {http://www.numdam.org/item/AIHPA_1986__44_3_263_0/}
}

Reply:
I'll just link to ADS.